locale/en/manageiq.po

```diff
@@ -37133,16 +37145,14 @@ msgid ""
 "y importing by skiping this playbook using --skip_playbook pb1, pb2."
 msgstr ""
 
-msgid "Playbook (AWX)"
-msgid_plural "Playbooks (AWX)"
+msgid "Playbook (%{provider_description})"
+msgid_plural "Playbooks (%{provider_description})"
 msgstr[0] ""
 msgstr[1] ""
 
 #, fuzzy
 msgid "Playbook (Ansible Tower)"
-msgid_plural "Playbooks (Ansible Tower)"
-msgstr[0] ""
-msgstr[1] ""
+msgstr ""
```

Comment:
@agrare This is interesting - I was expecting AnsibleTower to go away, but it appears some still exist in https://github.com/ManageIQ/manageiq/blob/master/locale/en.yml